orchestrator/gorc/src/commands/keys/cosmos/show.rs

```diff
@@ -0,0 +1,22 @@
+use abscissa_core::{Command, Options, Runnable};
+use signatory::FsKeyStore;
+use std::path::Path;
+
+#[derive(Command, Debug, Default, Options)]
+pub struct ShowCosmosKeyCmd {
+    #[options(short = "n", long = "name", help = "show [name]")]
+    pub name: String,
+}
+
+/// The `gorc keys cosmos show [name]` subcommand: show keys
+impl Runnable for ShowCosmosKeyCmd {
+    fn run(&self) {
+        // todo(shella): glue with signatory crate to list keys
+        let keystore_path = Path::new("/tmp/keystore");
+        let keystore = FsKeyStore::create_or_open(keystore_path).unwrap();
+        let key_name = &self.name.parse().unwrap();
+        let key_info = keystore.info(&key_name).unwrap();
+        let show_key = FsKeyStore::info(&keystore, &key_name).unwrap();
+        println!("{:?}", show_key)
+    }
+}
```

Comment:
Looking good! Some general feedback:

* We can use the app's config to locate the keystore now. Here's what that looks like now:

```rust
        let config = APP.config();
        let keystore = path::Path::new(&config.keystore);
        let keystore = signatory::FsKeyStore::create_or_open(keystore).unwrap();
```

* This command is using named flags instead of positional arguments. Some snippets you might find helpful for switching it over:

Here's what the name field on the struct evolves into:
```rust
    #[options(free, help = "show [name]")]
    pub args: Vec<String>,
```

And here's how it's consumed in `run`:
```rust
        let name = self.args.get(0).expect("name is required");
        let name = name.parse().expect("Could not parse name");
```

* We need to connect with Zaki to discuss _exactly_ what we we want this command to display. I suspect we need to load the private_key and display both it and the public_key with correct prefixes.

Reply:
We could fix a meeting for after standup